soroban-env-host/src/budget.rs

```diff
@@ -233,19 +233,19 @@ impl Default for BudgetImpl {
                 // whatever function call, arg-shuffling, spills, reloads or
                 // other flotsam accumulates around a typical memory copy.
                 ContractCostType::MemCpy => {
-                    cpu.const_term = 250;
+                    cpu.const_term = 50;
```

Comment:
Why did this change (given that the comments suggests that the value isn't based on calibration results)?

Reply:
The linear term isn't calibration-derived, but the constant one is.

Reply:
That's right. The const term was previously derived from calibration. I just updated the number to the new calibration.